charts/global-network-policies/templates/allow-to-dns.yaml

```diff
@@ -15,6 +15,10 @@ spec:
       networking.gardener.cloud/to-dns: allowed
   egress:
   - to:
+    - ipBlock:
+        cidr: 100.64.0.10/32 # required for node local dns
+    - ipBlock:
+        cidr: 169.254.20.10/32 # required for node local dns
```

Comment:
Can you explain more about it, i.e., why are those particular two ranges required?